apps/test/unit/templates/instructions/RubricFieldTest.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import {UnconnectedRubricField as RubricField} from '@cdo/apps/templates/instructions/RubricField';
+
+const DEFAULT_PROPS = {
+  showFeedbackInputAreas: true,
+  rubricLevel: 'exceeds',
+  rubricValue: 'exceeded expectations',
+  disabledMode: false,
+  onChange: () => {},
+  currentlyChecked: false
+};
+
+describe('RubricField', () => {
+  it('Renders with the correct values', () => {
+    const wrapper = shallow(<RubricField {...DEFAULT_PROPS} />);
+
+    // Radio Button
+    const confirmCheckedRadioButton = wrapper.find('CheckedRadioButton').at(0);
+    expect(confirmCheckedRadioButton.props().checked).to.equal(false);
+
+    // Details
+    const confirmDetails = wrapper.find('details').at(0);
+    expect(confirmDetails).to.contain('Exceeds');
+    expect(confirmDetails).to.contain('exceeded expectations');
+  });
+  it('When showFeedbackInputAreas is false, there should not be a CheckedRadioButton', () => {
+    const wrapper = shallow(
+      <RubricField {...DEFAULT_PROPS} showFeedbackInputAreas={false} />
+    );
+    expect(wrapper.find('CheckedRadioButton')).to.have.lengthOf(0);
+  });
+  it('When that RubricField is the currently checked one', () => {
+    const wrapper = shallow(
+      <RubricField {...DEFAULT_PROPS} currentlyChecked={true} />
+    );
+
+    const confirmCheckedRadioButton = wrapper.find('CheckedRadioButton').at(0);
+    expect(confirmCheckedRadioButton.props().checked).to.equal(true);
+  });
+  it('Disable mode - the Checked button also disabled ', () => {
+    const wrapper = shallow(
+      <RubricField {...DEFAULT_PROPS} disabledMode={true} />
+    );
+
+    const confirmCheckedRadioButton = wrapper.find('CheckedRadioButton').at(0);
+    expect(confirmCheckedRadioButton.props().disabledMode).to.equal(true);
+  });
+  it('When hover, a border is added and tooltip shows', () => {
+    // not sure how to do this
```

Comment:
You're testing a version of this component that isn't Radium-wrapped, so I'm not sure this is possible as-is.  It may be no problem to export the wrapped component and test that.

I'd expect this to be something like:
```jsx
const wrapper = shallow(<RubricField {...DEFAULT_PROPS}/>);
wrapper.simulate('mouseenter', {/* not sure what event params are needed */});
// ...assert style change and tooltip showing...
```
...but would have to spend more time working out the details.

Reply:
It didn't like me just importing the wrapper component. I can dig in a little more on this but for now I think I'm gonna try to keep moving forward.